runtime/src/lib.rs

```diff
@@ -46,8 +46,7 @@ use frame_system::{
 	EnsureSigned,
 };
 pub use pallet_balances::Call as BalancesCall;
-use pallet_chain_extension_assets::weights::SubstrateWeight;
-pub use pallet_chain_extension_assets::weights::WeightInfo as ChainExtensionWeightInfo;
+// pub use pallet_chain_extension_assets::weights::WeightInfo as ChainExtensionWeightInfo;
```

Comment:
Again why removing chain extensions here?

Reply:
`WeightInfo` became private in `pallet_chain_extension_assets` and it is not used in `runtime/lib.rs`